### DISCUSSION TITLE
Welcome to TeleDart Discussions!

### DISCUSSION THREAD

**Discussion by DinoLeung** [DinoLeung/TeleDart#142] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zaidite** [comment]
Hi, Dino!
Are you planning to create a tutorial on using Teledart in the near future?
It would be very cool if Teledart had instructions for use. Description of its concept and all the possibilities.

**Comment by zaidite** [comment]
How can I handle webhooks?
I have installed a webhook, but I cannot receive messages when they write to a chatbot or in a chat where there is a chatbot.
Can I ask for an example of receiving and processing messages when the webhook is enabled?

**Reply by DinoLeung** [reply]
Hi @zaidite please check out the repo below, it's a working example of using webhook.

https://github.com/DinoLeung/DartWiseBot/blob/master/bin/server.dart

**Reply by zaidite** [reply]
Thank you so much! Exactly what is needed!

**Comment by zaidite** [comment]
Hi, Dino!
Please tell me how to handle an event in the chat - adding / removing a bot?
Now I just send OK 200 and Telegram starts sending me many identical requests ...
I cannot simply teledart.replyMessage, since there was no message, but the bot was removed from the chat.
Or am I misunderstanding something?

Please give me an example of how to properly handle deleting and adding a bot to a chat.

Thank you in advance for your response!

**Reply by DinoLeung** [reply]
would you be able to give out more details about things you are trying to achieve? as in what sort of action you would like to perform after the bot being added or removed?
